src/rules.ts

```diff
@@ -501,7 +501,7 @@ export const rules: Rule[] = [
 
           for (const line of insert_lines) {
             const key = line.split(':')[0];
-            if (!parsed_yaml.hasOwnProperty(key)) {
+            if (!Object.hasOwnProperty.call(parsed_yaml, key)) {
```

Comment:
I'm not a javascript expert, why do you use Object.hasOwnProperty?

Reply:
I am not super familiar in TS/JS either, but ESLint complained. I believe this is because it is considered 'bad' to assume an object inherits from the Object class and thus has a `hasOwnProperty` method.

There is a short explanation on the relevant ESLint rule: https://eslint.org/docs/rules/no-prototype-builtins